engines/gob/sound/sounddesc.cpp

```diff
@@ -161,18 +161,21 @@ bool SoundDesc::loadWAV(byte *data, uint32 dSize) {
 
 	int wavSize, wavRate;
 	byte wavFlags;
-	uint16 wavtype;
+	uint16 wavType;
+	int numChannels; 
```

Comment:
We won't be needing this, as its usage is handled internally in `loadWAVFromStream()`

Reply:
If I understand correctly remove numChannels and use nullptr in the bool?

Reply:
> If I understand correctly remove numChannels

Yes